stories/carousel/carousel.stories.js

```diff
@@ -0,0 +1,480 @@
+                <li 
+                    data-slide-to="4" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="5" 
+                    aria-label="Displaying item 5 of 7" 
+                    class="fd-carousel__page-indicator fd-carousel__page-indicator--active"></li>
+                <li 
+                    data-slide-to="6" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="7" 
+                    class="fd-carousel__page-indicator"></li>
+            </ol>
+            <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next"
+                aria-label="Go to next item">
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-1" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+
+    <h4>Hiding Navigation Buttons in the Content</h4>
+    <div 
+        class="fd-carousel fd-carousel--no-navigation" 
+        data-ride="carousel" 
+        style="margin-bottom: 3rem; max-width: 30rem;">
+        <div class="fd-carousel__page-indicator-container">
+            <ol class="fd-carousel__page-indicators">
+                <li 
+                    data-slide-to="1" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="2" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="3" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="4" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="5" 
+                    class="fd-carousel__page-indicator fd-carousel__page-indicator--active"></li>
+                <li 
+                    data-slide-to="6" 
+                    class="fd-carousel__page-indicator"></li>
+                <li 
+                    data-slide-to="7" 
+                    class="fd-carousel__page-indicator"></li>
+            </ol>
+        </div>
+        <div class="fd-carousel__content" style="min-height: 15.5rem;">
+           <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+            </button>
+           <button 
+                class="fd-button fd-carousel__button sap-icon--slim-arrow-right"
+                data-slide="next"
+                aria-label="Go to next item">
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-6" aria-live="polite">
+        Displaying item 1 of 4
+    </div>
+
+</div>
+`;
+
+carouselNoNavigation.storyName = 'Hiding the Navigation Buttons';
```

Comment:
"Carousel can also be displayed without navigation buttons. To hide them, add the` fd-carousel--no-navigation` modifier class to the `fd-carousel` class. On touchable devices, the user can navigate with a swipe gesture."